docs/Config.md

```diff
@@ -75,3 +75,45 @@ The secrets in this example are dummy values and won't work in an actual app.
   }
 }
 ```
+
+### Running over HTTPS
+
+You need to change change the `protocol` to `https`, and tell the server where
+to find the certificate and the private key (`crt_file` and `key_file`). You'll
+need to add these values to both the `asset_server` and the `app_server`.
+
+```json
+{
+  "api": {
+    "protocol": "https",
+    "host": "www.api.dev.meetup.com",
+    "timeout": 10,
+    "root_url": "https://www.api.dev.meetup.com"
+  },
+  "asset_server": {
+    "protocol": "https",
+    "host": "beta2.dev.meetup.com",
+    "port": 0,
+    "crt_file":"/path/to/certificate.crt",
```

Comment:
`crt_file` and `key_file` are attached to both `asset_server` and `app_server`. I like that they are in the same place we're setting the protocol, but it's almost always going to be a duplicate config.